packages/ui-components/package.json

```diff
@@ -13,7 +13,8 @@
     "clean": "rm -rf  ./dist/*",
     "lint": "eslint ./src/**/*.{tsx,ts,js} --format=codeframe",
     "test": "jest --watch",
-    "watch": "webpack"
+    "watch": "webpack",
```

Comment:
I know you didn't add this line but should it be `webpack --watch`?

Reply:
Maybe also unify with the icons package and change the name to `build:watch` since `start` will simplify it anyway.

Reply:
`webpack` command in ui-components starts webpack dev server. and `build:watch` seems to actually create dist files on every change. So I would not rename them as this commands already sorta correctly named to be self explanatory.

Reply:
We don't have `webpack-dev-server` installed but it looks like watch is set to true in the webpack config https://github.com/cockroachdb/ui/blob/master/packages/ui-components/webpack.config.js#L7

Reply:
I think removing `watch` from the config and then updating `"watch": "webpack"` to be `"build:watch": "webpack --watch"` would be the most clear here. It's building when files change like the `ui-components` repo. Does that make sense to you @elkmaster?